src/widgets/programs/programs-detailed-summary.component.tsx

```diff
@@ -27,115 +28,101 @@ export default function ProgramsDetailedSummary(
-                            </svg>
-                          </Link>
-                        }
-                      </td>
-                    </tr>
-                  </React.Fragment>
-                );
-              })}
-          </tbody>
-        </table>
-      </SummaryCard>
-    );
-  }
-
-  function displayNoPrograms() {
-    return (
-      <SummaryCard
-        name="Care Programs"
-        styles={{
-          width: "100%",
-          background: "var(--omrs-color-bg-low-contrast)",
-          border: "none",
-          boxShadow: "none"
-        }}
-      >
-        <div className={styles.emptyPrograms}>
-          <p data-testid="no-programs">
-            Program data will appear here once the patient enrolls into a
-            program.
-          </p>
-          <p>
-            <button
-              className="omrs-btn omrs-outlined-action"
-              onClick={() => openWorkspaceTab(ProgramsForm, "Programs Form")}
-            >
-              Enroll in a program
-            </button>
-          </p>
-        </div>
-      </SummaryCard>
-    );
-  }
+  }, [patientUuid, isLoadingPatient]);
 
   return (
     <>
-      {enrolledPrograms && (
+      {enrolledPrograms && enrolledPrograms.length > 0 ? (
         <div className={styles.programsSummary}>
-          {enrolledPrograms.length > 0
-            ? displayPrograms()
-            : displayNoPrograms()}
+          <SummaryCard
+            name={t("Care Programs")}
+            styles={{
+              width: "100%"
+            }}
+            addComponent={ProgramsForm}
+            showComponent={() =>
+              openWorkspaceTab(ProgramsForm, `${t("Programs Form")}`)
+            }
+          >
+            <table className={`omrs-type-body-regular ${styles.programTable}`}>
+              <thead>
+                <tr>
+                  <th>
+                    <Trans i18nKey="activePrograms">Active Programs</Trans>
+                  </th>
+                  <th>
+                    <Trans i18nKey="since">Since</Trans>
+                  </th>
+                  <th>
+                    <Trans i18nKey="status">Status</Trans>
+                  </th>
+                  <th></th>
+                </tr>
+              </thead>
+              <tbody>
+                {enrolledPrograms?.map(program => {
+                  return (
+                    <React.Fragment key={program.uuid}>
+                      <tr
+                        className={`${
+                          program.dateCompleted
+                            ? `${styles.inactive}`
+                            : `${styles.active}`
+                        }`}
+                      >
+                        <td className="omrs-medium">{program.display}</td>
+                        <td>
+                          {dayjs(program.dateEnrolled).format("MMM-YYYY")}
+                        </td>
+                        <td>
+                          {program?.dateCompleted ? (
```

Comment:
Inconsistent - in this branch we know (and use) program already non-conditionally, e.g., see `program.display` above. No need for the conditional member access here.